charts/karmada/templates/_karmada_webhook_configuration.tpl

```diff
@@ -199,4 +199,18 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: ["v1"]
     timeoutSeconds: 3
+  - name: multiclusterservice.karmada.io
+    rules:
+      - operations: ["CREATE", "UPDATE"]
+        apiGroups: ["networking.karmada.io"]
+        apiVersions: ["*"]
+        resources: ["multiclusterservices"]
+        scope: "Namespaced"
+    clientConfig:
+      url: https://{{ $name }}-webhook.{{ $namespace }}.svc:443/validate-multiclusterservice
+      {{- include "karmada.webhook.caBundle" . | nindent 6 }}
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions: [ "v1" ]
+    timeoutSeconds: 3
```

Comment:
here is 3s, `artifacts/deploy/webhook-configuration.yaml` is 10s, can we keep them same?

Reply:
I'm also a little confused here, for the `timeoutSeconds` of webhook configuration in [charts/karmada/templates/_karmada_webhook_configuration.tpl](https://github.com/karmada-io/karmada/pull/3779/files/b5280dcf71280b1ad5a90cb9dec0c31d55cdf9c7#diff-18e1d51f488df0df7ce80b3131d3445be73a31bde39a7c36ce42c71e940a8a47) is 3s, so I make it 3s here.
And we can see that `timeoutSeconds` is different among configuration files, such as `artifacts/deploy/webhook-configuration.yaml`, `charts/karmada/templates/_karmada_webhook_configuration.tpl`.

Reply:
Let's set all of them to 3s. Is that ok? @chaunceyjiang @XiShanYongYe-Chang

Reply:
In #2237 , the timeout duration is extended to 10s. It's been a while, maybe we can first see what happens when the timeout duration is changed to 3s. If the problem with the CI environment has been fixed, we keep the timeout duration 3s. And if the CI has frequent errors later on, we can then set the timeout duration more leniently. Is this ok? @XiShanYongYe-Chang @jwcesign @chaunceyjiang

Reply:
ok. I agree with it, thx.